Applications/WebPortal/WebApp/library/source/pages/page_not_found_page.tsx

```diff
@@ -0,0 +1,89 @@
+import { HBoxLayout, Padding, VBoxLayout } from 'dali';
+import * as React from 'react';
+import { HLine } from '../components';
+import { DisplaySize } from '../display_size';
+import { PageWrapper } from './page_wrapper';
+
+interface Properties {
+  
+  /* The size of the viewport. */
+  displaySize: DisplaySize;
+}
+
+/* Displays a page that indicates the desired page cannot be found. */
+export class PageNotFoundPage extends React.Component<Properties> {
+
+  public render(): JSX.Element {
+    const headerTextStyle = (() => {
+      if(this.props.displaySize === DisplaySize.LARGE) {
+        return PageNotFoundPage.STYLE.headerTextLarge;
```

Comment:
Use the new "array" like style you used in a previous pull request. ```PageNotFoundPage.Style[this.props.displaySize]```. You can then inline the style into the component instead of initializing it up here.